.github/workflows/erlang.yml

```diff
@@ -24,6 +24,10 @@ jobs:
       with:
         path: /tmp/.buildx-cache
         key: ${{ runner.os }}-buildx-${{ hashFiles('./example/image/Dockerfile') }}
+    - name: Install docker-compose
+      run: |
+        curl -L "https://github.com/docker/compose/releases/download/1.29.2/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose
```

Comment:
docker-compose doesn't seemed to be install anymore (although a plugin is), looking it up on docker github, this was the solution.